MekHQ/src/mekhq/campaign/autoresolve/acar/action/ManeuverToHitData.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 - The MegaMek Team. All Rights Reserved.
+ *
+ *  This file is part of MekHQ.
+ *
+ *  MekHQ is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  MekHQ is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with MekHQ. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package mekhq.campaign.autoresolve.acar.action;
+
+import megamek.common.TargetRoll;
+import mekhq.campaign.autoresolve.component.EngagementControl;
+import mekhq.campaign.autoresolve.component.Formation;
+
+public class ManeuverToHitData extends TargetRoll {
+
+    public ManeuverToHitData(int value, String desc) {
+        super(value, desc);
+    }
+
+    public static ManeuverToHitData compileToHit(Formation formation) {
+        var toHit = new ManeuverToHitData(formation.getTactics(), "Tactics");
+        processFormationModifiers(toHit, formation);
+        processCombatUnit(toHit, formation);
+        return toHit;
+    }
+
+    private static void processFormationModifiers(ManeuverToHitData toHit, Formation formation) {
+        if (formation.getEngagementControl() == EngagementControl.FORCED_ENGAGEMENT) {
+            toHit.addModifier(1, "Forced Engagement");
+        }
+        if (formation.isAerospace()) {
+            toHit.addModifier(2, "Aerospace Formation");
+        }
+    }
+
+    private static void processCombatUnit(ManeuverToHitData toHit, Formation formation) {
```

Comment:
Would the SkillLevel enums be a better fit here, rather than Strings? 

e.g. `SkillLevel.ELITE.toString()` or `SkillLevel.ELITE.name()`

Reply:
Not really, its not a 1:1 match with SkillLevel